SpatialGDK/Source/SpatialGDKFunctionalTests/Public/ReplicatedTestActorBase.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "GameFramework/Actor.h"
+#include "ReplicatedTestActorBase.generated.h"
+
+/**
+ * A replicated Actor with a Cube Mesh, used as a base for Actors used in tests.
+ */
+UCLASS()
+class AReplicatedTestActorBase : public AActor
```

Comment:
Oh, yes, I will move this class, I thought that since probably other classes from test folders will inherit for this and therefore include this class it would be clean to have it here, but probably creating a folder inside the SpatialGDK such as BaseActors and adding it there is a better call.